SYN-9180: synapse.tools.cortex.layer.{dump, load} (#4325)
vertexproject/synapse: docs/synapse/devopsguide.rst

tools available to export nodes edits from a layer on the source Cortex and then
import those same edits into a layer on a different Cortex.

- ``synapse.tools.cortex.layer.dump``: Export node edits from a specified layer on a
  Cortex.
- ``synapse.tools.cortex.layer.load``: Import node edits to a specified layer on a
  Cortex.

In these situations, an export would be created using
``synapse.tools.cortex.layer.dump``. The output of this tool is one or more export
files that can then be manually copied over to the destination Cortex
environment. Once the files have been copied, the ``synapse.tools.cortex.layer.load``
tool can be used to import them into the destination Cortex.

.. note::

   The ``synapse.tools.cortex.layer.dump`` export tool creates a state tracking YAML
   file for recording the next expected offset from the layer that was exported.
   The state tracking files allows the export tool to create incremental exports
   from the time of the last execution without requiring the offset to be
   separately tracked.

   The filename for the state tracking file is automatically saved in the output
   directory and follows the name ``<celliden>.<layriden>.yaml`` but can be

>    specified using the ``--statefile`` command line option.

Reviewer: ```suggestion
   overridden using the ``--statefile`` command line option to specify an alternate path.
```

Export node edits from a Cortex
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Exporting node edits from a layer can be done with the
``synapse.tools.layer.dump`` command::

>     python -m synapse.tools.layer.dump <layriden>

Reviewer 2: ```suggestion
    python -m synapse.tools.layer.dump --url <cortexurl> <layriden>
```

When running the export tool from within the Cortex container, the ``--url``

> option does not need to be provided as it will default to ``/vertex/storage``.

Reviewer 2: ```suggestion
option does not need to be provided as it will default to the local service.
```
